nix/default.nix

```diff
@@ -113,6 +116,11 @@ in
         }'
       '';
 
+      postInstall = ''
+        wrapProgram $out/bin/Hyprland \
+          --suffix PATH : ${lib.makeBinPath [ binutils ]}
```

Comment:
I think this would also need `pciutils`

Reply:
Wasn't sure if i should add it since its less of a hard dependency than binutils. Only used for printing stats on launch and in the crash report

Reply:
pciutils is currentlyin bulidinputs, not sure why it's there, don't think it's doing anything..

Reply:
Probably lspci

Reply:
it needs lspci in PATH at runtime(like nm). adding it to buildinputs doesn't do that